src/project/FileViewController.js

```diff
@@ -70,14 +69,6 @@ define(function (require, exports, module) {
      */
     var _fileSelectionFocus = PROJECT_MANAGER;
 
-    // Due to circular dependencies, not safe to call on() directly
-    /**
-     * Change the doc selection to the working set when ever a new file is added to the working set
-     */
-    EventDispatcher.on_duringInit(MainViewManager, "workingSetAdd", function (event, addedFile) {
-        _fileSelectionFocus = WORKING_SET_VIEW;
```

Comment:
I think you should just change this line to the following (NOTE: the comment too):

```js
// XXXBramble: always keep focus on the Project Manager vs. Working Set View
_fileSelectionFocus = PROJECT_MANAGER;
```

If that works OK, I think it's better to keep the events in place.  If that doesn't work, we can explore removing this.